tests/test_check_addon.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from kodi_addon_checker import _all_repo_addons
+from kodi_addon_checker.check_addon import start
+from kodi_addon_checker.record import Record
+from kodi_addon_checker.config import Config
+
+
+class TestCheckAddon(unittest.TestCase):
+    """Integration tests for Start function present in check_addon.py"""
+
+    def setUp(self):
+        self.path = "script.test/"
+        self.whitelist = ["INFO: checking addon", "INFO: created by",
+                          "Image icon", "Icon dimensions are fine", "INFO: Image fanart",
+                          "WARN: Complex entry point", "WARN: We found", "please check the logfile"]
+        self.config = Config(self.path)
+        self.branch_name = "krypton"
+        self.all_repo_addons = _all_repo_addons()
+
+    def test_start(self):
+        result = start(self.path, self.branch_name, self.all_repo_addons, self.config)
+        values = [Record.__str__(r) for r in result]
+        self.assertTrue(any(any(x in z for z in values) for x in self.whitelist))
```

Comment:
can you try to use variables that are not single characters?

Reply:
like should I change them within comprehension or should I change this line into multiple lines ?